tests/acceptance/features/apiSearch/dateSearch.feature

```diff
@@ -39,7 +39,9 @@ Feature: date search
       | Mtime:yesterday    | /yesterday.txt  |                 | /today.txt      |                 |
       | Mtime:"this week"  | /today.txt      |                 | /lastWeek.txt   | /lastMont.txt   |
       | Mtime:"this month" | /today.txt      |                 | /lastMont.txt   |                 |
-      | Mtime:"last month" | /lastMonth.txt  |                 | /today.txt      |                 |
+      # Issue: https://github.com/owncloud/ocis/issues/7629
+      # uncomment when issue is fixed
+      # | Mtime:"last month" | /lastMonth.txt  |                 | /today.txt      |                 |
```

Comment:
Can't we use the expected failures file for this? Then we don't forget to reenable it once the fix landed. Not critical though....

Reply:
I am expecting that it will pass tomorrow onward

Reply:
date 31 is the tricky one